mdpow/fep.py

```diff
@@ -168,6 +176,11 @@ def kJ_to_kcal(x):
     """Convert a energy in kJ to kcal."""
     return x / 4.184
 
+def kBT_to_kJ(x, T):
+    """Convert a energy in kBT to kJ/mol."""
+    x *= constants.N_A*constants.k*T*1e-3
+    return x
```

Comment:
```suggestion
    return x * constants.N_A*constants.k*T*1e-3
```
I think it's safer to always return a copy (if x is an array) instead of changing it in place.